sous_chef/sous_chef/messaging/todoist_api.py

```diff
@@ -302,7 +303,7 @@ def _get_active_labels(self):
         return label_dict
 
     @staticmethod
-    def _get_due_date_str(due_date: datetime.datetime) -> str:
+    def _get_due_date_str(due_date: datetime.datetime) -> Union[str, None]:
```

Comment:
as discussed, there should be a check or something that ensures only stings go here.

Reply:
Changed that behaviour and corresponding tests, pls have a look.